src/main/java/com/tileman/TilemanModeConfig.java

```diff
@@ -94,14 +97,37 @@ default int warningLimit() {
         return 50;
     }
 
+    @ConfigItem(
+            keyName = "enableTilesWarning",
+            name = "Enable Tiles Warning",
+            section = settingsSection,
+            description = "Turns on tile warnings when you reach your set limit or 0.",
+            position = 4
+    )
+    default boolean enableTileWarnings() {
+        return true;
+    }
+
+    @ConfigItem(
```

Comment:
Why allow this?

Reply:
Think a couple people asked for it, could just remove the config and allow people to manually mark into the negatives